azurelinuxagent/common/protocol/wire.py

```diff
@@ -1591,11 +1591,6 @@ def parse_plugin(self, plugin):
         ext_handler.properties.version = getattrib(plugin, "version")
         ext_handler.properties.state = getattrib(plugin, "state")
 
-        try:
-            ext_handler.properties.dependencyLevel = int(getattrib(plugin, "dependencyLevel"))
-        except ValueError:
-            ext_handler.properties.dependencyLevel = 0
-
```

Comment:
I think we also need to remove the initialization of dependencyLevel in ExtHandlerProperties.__init__

Reply:
Removed it.